src/vip/batch_address_test_tool/processor.clj

```diff
@@ -63,7 +63,10 @@
 
 (defn retrieve-polling-locations*
   [ctx]
-  ctx)
+  (let [addresses (:addresses ctx)
+        polling-locations (doall (map #(civic-info/address->polling-location (:address %)) addresses))
```

Comment:
Maybe writing a test that demonstrates what's doing on would help my tired brain.